TESTS/mbedmicro-rtos-mbed/mutex/main.cpp

```diff
@@ -142,7 +142,7 @@ void test_dual_thread_lock_lock_thread(Mutex *mutex)
     uint32_t start = us_ticker_read();
 
     osStatus stat = mutex->lock(TEST_HALF_SEC_MS);
-    TEST_ASSERT_EQUAL(stat, osEventTimeout);
+    TEST_ASSERT_EQUAL(stat, osErrorTimeout);
```

Comment:
Can the event codes (osEventTimeout) be typedef or defined as error codes (osErrorTimeout) so RTX5 is more backwards compatible?